src/monodroid/jni/android-system.cc

```diff
@@ -300,20 +310,21 @@ AndroidSystem::_monodroid_get_system_property_from_file (const char *path, char
 		return 0;
 	}
 
+	size_t file_size = static_cast<size_t>(fileStat.st_size);
```

Comment:
Why `size_t` and not `off_t`?

Reply:
`size_t` is unsigned, `off_t` is signed and files cannot have a negative size